syntax = "proto3";
package dfi;

message DFIRetrieveBufferRequest {
	string buffername = 1;
	string flowname = 2;
	bool isSource = 3;
	bool isTarget = 4;
    uint64 targetid = 5;
}
